src/tests/containers/CreateProjectPage.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { CreateProjectPage } from '../../containers/CreateProjectPage'
+
+describe('CreateProjectPage', () => {
+  let wrapper
+  let props = {
+    history: { push: jest.fn() },
```

Comment:
do we need this, or was this added when we were thinking of handling the redirection from the `CreateProjectPage.js`?